playbook/app/pb_kits/playbook/pb_advanced_table/_advanced_table.scss

```diff
@@ -215,11 +215,13 @@
           background-color: $white;
       }
 
-    }
-  }
-  @media only screen and (min-width: $screen-xl-min) {
-    &[class*="advanced-table-responsive-scroll"] {
-      overflow-x: visible;
+      .row-selection-actions-card {
+        border-right-width: 0px;
+        position:sticky;
```

Comment:
Hey Nida! You are missing a space before sticky!

Reply:
thanks Nick!!!